Source/WebCore/page/CaptionUserPreferencesMediaAF.cpp

```diff
@@ -721,8 +721,8 @@ static String trackDisplayName(const TrackBase& track)
 
     auto preferredLanguages = userPreferredLanguages(ShouldMinimizeLanguages::No);
     auto defaultLanguage = !preferredLanguages.isEmpty() ? preferredLanguages[0] : emptyString(); // This matches `defaultLanguage`.
-    auto currentLocale = adoptCF(CFLocaleCreate(kCFAllocatorDefault, defaultLanguage.createCFString().get()));
-    auto localeIdentifier = adoptCF(CFLocaleCreateCanonicalLocaleIdentifierFromString(kCFAllocatorDefault, trackLanguageIdentifier.createCFString().get()));
+    RetainPtr currentLocale = adoptCF(CFLocaleCreate(kCFAllocatorDefault, defaultLanguage.createCFString().get()));
```

Comment:
I think implicit in this change, and the many like it, is a style guideline: Write `RetainPtr` explicitly to make the object lifetime more obvious than if you wrote `auto`. Presumably similar for other types. The main reason I haven’t been a fan of that rule in the past is that it doesn’t work when you don’t have a named local variable, and I often like the brevity of doing `adoptCF` in an expression in a function call without having to name the result, and other similar idioms. I figured that if we had a good quality checking system it would be just as easy to get it right without having to see the name of the type. But I am OK with us going this way, especially if it’s helping us do this better in practice.